code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1276,9 +1276,10 @@
 	else
 		update_icon_dropped()
 
+// Note: Does NOT return EMP protection value from parent call or pass it on to subtypes
```

Comment:
man this is so wack looking at the code, you sure this is the best way? can't we just have a new bitflag with something like `EMP_ACTION_SUCCEEDED` that we listen for on subtypes? do other subtypes futz with `emp_act()` this way? i'd rather keep it all bitflags rather than just be weird and inconsistent

Reply:
The more sensible option is to have EMP act pass the protection as an argument rather than via parent call but that's a whole thing

Reply:
yeah that would make the most sense, too bad it's out of scope for this PR. i still am not a fan of it but it's clear that without some major overhaul to this proc that the root issue is not gonna be fixed so